assets/router/deployment.yaml

```diff
@@ -31,19 +31,33 @@ spec:
           - name: DEFAULT_DESTINATION_CA_PATH
             value: /var/run/configmaps/service-ca/service-ca.crt
           livenessProbe:
+            failureThreshold: 3
             httpGet:
               path: /healthz
               port: 1936
+              scheme: HTTP
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+            terminationGracePeriodSeconds: 10
```

Comment:
Note that #867 added `terminationGracePeriodSeconds` for the liveness probe, so #1092 is effectively backporting part of #867.

Reply:
I think you meant #863.  I'll try to extricate it.

Reply:
Fixed - changes in https://github.com/openshift/cluster-ingress-operator/compare/190f2178b3f65f60a97d1e37f13dad15c3017bb0..511046a031933f0c5093b1a785af7e893a80e9f8